geniza/corpus/models.py

```diff
@@ -677,11 +683,12 @@ def related_documents(self):
 
     def has_transcription(self):
         """Admin display field indicating if document has a transcription."""
-        return any(note.has_transcription() for note in self.footnotes.all())
+        return self.footnotes.filter(
+            doc_relation__contains=Footnote.DIGITAL_EDITION
+        ).exists()
```

Comment:
This was implemented on the local versions of the footnotes to avoid an extra db call, since this property is used in the admin list view. (Although there should have been a comment to that effect!)

Please revise to implement it without an extra db call and add a comment about avoiding the extra db call for admin list view